components/size_aware_image.jsx

```diff
@@ -276,7 +276,7 @@ export default class SizeAwareImage extends React.PureComponent {
                     target='_blank'
                     rel='noopener noreferrer'
                     download={true}
-                    role={this.isInternalImage ? 'button' : false}
+                    role={this.isInternalImage ? 'button' : undefined}
```

Comment:
What does this mean for accessibility? Should this value be `link` instead of undefined

Reply:
I think that just means that it's treated as a regular link instead of a button since I'd assume `role` is a way to override whatever roles are implied by the HTML tag. I don't fully understand why we only set the `role` sometimes, but setting it to undefined is no worse than `false` at least 😛